frank-doc-doclet/src/main/java/org/frankframework/frankdoc/FrankDocJsonFactory.java

```diff
@@ -439,4 +442,46 @@ private Optional<JsonArray> getLabels() {
 		}
 		return Optional.of(result.build());
 	}
+
+	// The properties are optional, so they can be omitted from unit tests.
+	private Optional<JsonArray> getProperties() {
+		if (model.getPropertyGroups().isEmpty()) {
+			return Optional.empty();
+		}
+
+		var b = bf.createArrayBuilder();
```

Comment:
Ik weet niet wat de afspraak is over het gebruik van 'var' (ik heb het volgens mij nog niet gezien), maar ik zou wel voorstellen om dan een duidelijke naam voor de variabele te kiezen. Bv. `var arrayBuilder` ofzo.